### ISSUE TITLE
scorecard coefficients

### ISSUE THREAD

**Issue by jmiele6** [guillermo-navas-palencia/optbinning#233]
Thank you for this library it really helps a lot!

 I have to questions concerning scorecard:
1. Is it possible to add intercept to the scorecard?
2. The coefficients that I get from the scorecard are not the same as I would get from a logistic regression on binned variables (although they are close to them). Why is it so?

**Comment by guillermo-navas-palencia** [comment]
Hi @jmiele6,

1. Yes, it is possible. See: http://gnpalencia.org/optbinning/scorecard.html parameter ``intercept_based``.
2. Could you provide an example? make sure the ``scaling_method`` is set to None

**Comment by jmiele6** [comment]
Hi,
I noticed that the source of my problem was manual adjustment of bins, which was not propagated to scorecard, issue described in: 
https://github.com/guillermo-navas-palencia/optbinning/issues/151.
After using binning_fit_params I got bins, on which coefficients of the scorecard were the same as from regression on binned variables. Therefore the intecept which I needed from the scorecard could be taken from regression model.
